network_info_modem_info.c

```diff
@@ -122,7 +122,8 @@ int network_info_add_to_map(zcbor_state_t *response_detail_map)
 	return GOLIOTH_RPC_OK;
 
 rpc_exhausted:
-	LOG_ERR("Failed to encode value");
+	LOG_ERR("Failed to encode some values; limited by CONFIG_GOLIOTH_RPC_MAX_RESPONSE_LEN: %d",
```

Comment:
This depends on having `v0.11.1` or newer of the Golioth Firmware SDK. I'll make an update to the CHANGELOG that indicates this.

Reply:
In this context, can `zcbor_tstr_put_lit` fail to encode for reasons other than being limited by `CONFIG_GOLIOTH_RPC_MAX_RESPONSE_LEN`? I think it's helpful to have the reference to `CONFIG_GOLIOTH_RPC_MAX_RESPONSE_LEN` in the log, but just wondering if that might not always be the reason for the failure to encode (if that's the case, maybe just change the wording to reflect that it _may_ be limited by that?)